tests/core/atoms/conftest.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import gzip
+import shutil
+from pathlib import Path
+
+import pytest
+
+FILE_DIR = Path(__file__).resolve().parent
+
+
+def mock_generate_chemshell_cluster(*args, **kwargs):
+    # Copy the file to the output directory
+    with (
+        gzip.open(
+            f"{FILE_DIR}/skzcam_files/REF_ChemShell_cluster.xyz.gz", "rb"
+        ) as f_in,
+        open(f"{FILE_DIR}/ChemShell_cluster.xyz", "wb") as f_out,
```

Comment:
Please use `Path` objects instead of `str` concatenation for paths to ensure propery handling across filesystems.